apps/ensnode/src/handlers/NameWrapper.ts

```diff
@@ -158,8 +163,8 @@ export const makeNameWrapperHandlers = (ownedName: OwnedName) => {
       await upsertAccount(context, owner);
 
       await context.db.update(schema.domain, { id: node }).set((domain) => ({
-        // https://github.com/ensdomains/ens-subgraph/blob/master/src/nameWrapper.ts#L123
         // null expiry date if the domain is not a direct child of .eth
+        // via https://github.com/ensdomains/ens-subgraph/blob/master/src/nameWrapper.ts#L123
```

Comment:
It's a general hint: please refer to repositories contents via links including commit sha. Branches keep changing, commits remain the same 👍